includes/definitions/discovery/panos.yaml

```diff
@@ -32,9 +32,15 @@ modules:
         count:
             data:
                 -
-                    oid: panSession
-                    value: panSessionActive
-                    num_oid: '.1.3.6.1.4.1.25461.2.1.2.3.{{ $index }}'
+                    oid: panSessionActive
+                    num_oid: '.1.3.6.1.4.1.25461.2.1.2.3.3.{{ $index }}'
                     group: Sessions
                     descr: Active Sessions
-                    high_limit: panSessionMax
+                    high_limit: '.1.3.6.1.4.1.25461.2.1.2.3.2.{{ $index }}'
+        load:
```

Comment:
This should be a count sensor.

Load is for electrical AMPs.

Reply:
charge, humidity, and load were the only sensors listed here https://docs.librenms.org/Developing/os/Health-Information/ that supported receiving the value as a percentage. Count appeared to expect an integer or count of how many sessions there were. 

Does a new sensor type need to be added instead?

Reply:
According to the description, this is a count of the number of sessions, or am I missing something.

Reply:
Maybe we are talking about a different line? Description is "SSL Proxy Utilization" and is given as a percentage. The other health monitoring called "Active Sessions" is indeed a count and is under the count type.

Reply:
Load is ok, for that since it is a percentage.  I assumed incorrectly it was a count :)